### ISSUE TITLE
Newbie questions

### ISSUE THREAD

**Issue by Funionz** [kolton/d2bot-with-kolbot#1560]
So I just downloaded kolbot and was watching a video on how to set it up
The first thing the video said is to make sure not to run it with the game dl'd from blizzards.com
Will I get banned quickly if I don't torrent the game and a cdkey or is it fine now?
I used this bot several years ago on an official dl and never had problems
Also I saw some things about needing to have 20 min games to avoid ban, is this still relevant?
I plan on having passworded games simply for mf purposes if it matters at all

**Comment by DarkHorseDre** [comment]
no, install from cd image and you'll be fine (I just did total reinstall a couple weeks ago)

yeah 20 mins per account and cdkey i believe - you sound like you wont hit issues - just set minimum game time and also other crash/start settings to avoid too many games when d/c etc.

if pw game sp is easiest as you dont need to worry, but if you want bnet then you should be fine with the above - others may add anything I'm missing

**Comment by Funionz** [comment]
My bot kept a white superior great bow. Even with socket quest and 5os it seems like a bad faith base since bowazons would rather have a bow with skills in the base. Am I missing something?

**Comment by DarkHorseDre** [comment]
well without sharing your pickit or telling us whats in it I cant say.. obviously the pickit would need to state the skills you want to be in the base in order for it to keep that item..
